docs/fastdds/xml_configuration/domainparticipantfactory.rst

```diff
@@ -0,0 +1,86 @@
+.. include:: ../../03-exports/aliases.include
+.. include:: ../../03-exports/aliases-api.include
+.. include:: ../../03-exports/roles.include
+
+.. _participantfactoryprofiles:
+
+DomainParticipantFactory profiles
+---------------------------------
+
+The |DomainParticipantFactory| profiles allow the definition of the configuration of |DomainParticipantFactory-api|
+through XML files.
+These profiles are defined within the ``<domainparticipant_factory>`` XML tags.
+
+.. _domainparticipantfactoryattributes:
+
+DomainParticipantFactory XML attributes
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The ``<domainparticipant_factory>`` element has two attributes defined: ``profile_name`` and ``is_default_profile``.
+
+.. list-table::
+   :header-rows: 1
+   :align: left
+
+   * - Name
+     - Description
+     - Use
+   * - ``profile_name``
+     - Sets the name under which the ``<domainparticipant_factory>`` profile is registered in the |br|
+       DDS Domain, so that it can be loaded later by the |DomainParticipantFactory-api|, as shown in |br|
+       :ref:`loadingapplyingprofiles`.
+     - Mandatory
+   * - ``is_default_profile``
+     - Sets the ``<domainparticipant_factory>`` profile as the default profile. Thus, if a default |br|
+       profile exists, it will be used when creating the DomainParticipantFactory
+     - Optional
```

Comment:
Does it make sense for this to exist? Shouldn't it always be `true`

Reply:
Let's discuss this internally